runtime/src/accounts_db.rs

```diff
@@ -634,6 +642,18 @@ impl AccountsDB {
     // can be purged because there are no live append vecs in the ancestors
     pub fn purge_zero_lamport_accounts(&self, ancestors: &HashMap<u64, usize>) {
```

Comment:
This is trivial, but I think making this checking code programmatic and adding a test for it is desirable by wrapping the existing code like this:

```
-   pub fn purge_zero_lamport_accounts(&self, ancestors: &HashMap<u64, usize>)
+   pub fn do_purge_zero_lamport_accounts(&self, ancestors: &HashMap<u64, usize>)
...

+ pub fn (try_to_)purge_zero_lamport_accounts(&self, ancestors: &HashMap<u64, usize>) {
   if self.is_zla_is_increased() { // <= unit test this newly added predicate
        self.do_purge_zero_lamport_accounts(ancestors);
   }
}
```